modifiers/hidescore.css

```diff
@@ -0,0 +1,3 @@
+#score {
+	display: none;
+}
```

Comment:
```suggestion
}

```

Reply:
This change request comes from this convention:
https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_206

But @Psycast points out that the rest of the .css files in the repo don't have this convention, so there's no point in adding it in this submission.